### PR TITLE
data:MichaelLarocca

### PR DESCRIPTION
**Pull request by MichaelLarocca**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

My first pull request to create a LinkFree profile

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4071"></a>

### CODE REVIEW

File: data/MichaelLarocca.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  ],
+  "socials": [
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/michael-larocca-9315a3202/" },
+    { "icon": "FaTwitter", "url": "https://twitter.com/MikeJudeLarocca" },
+    { "icon": "FaGithub", "url": "https://github.com/MichaelLarocca" }
+  ],
+  "testimonials": [
+    "HOLD",
+    "HOLD",
+    "HOLD"
```
Comment (eddiejaoude) [inline data/MichaelLarocca.json:21]: This property for `testimonials` can be removed if not being used

Comment (MichaelLarocca) [reply data/MichaelLarocca.json:21]: Ok. I created a template to make the first pull request easier. I will remove it and add it back later when appropriate. 👍🏻

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Sindhuinti — APPROVED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**